docker-compose.yml

```diff
@@ -47,6 +47,22 @@ services:
       # port 5050 for the web interface (port 80 in the container).
       - "${PGADMIN_PORT:-5050}:80"
     restart: unless-stopped
+  inventory-management:
+    build: ./inventory-management
+    container_name: "inventory-management"
+    # Forward ports, format 'container:host'
+    ports:
+      - 3020:3020
+    # Mount local directories on the container filesystem.
+    # Should not happen in production.
+    volumes:
```

Comment:
You also need to add the tests directory as a bind mount, so that it's possible to run the tests in the container.